src/main/resources/db/migration/V1__create_customer_table.sql

```diff
@@ -0,0 +1,7 @@
+create TABLE customer (
+    id SERIAL PRIMARY KEY,
+    first_name varchar(50) not null,
+    last_name varchar(50) not null,
+    email varchar(50) not null unique,
+    password varchar(50) not null
+);
```

Comment:
What is this for ?

Reply:
This is related to our script for sharing the DB, it runs the first time someone new uses the application so they can also use the table